tests/validation/tests/v3_api/scripts/provision-config.yaml

```diff
@@ -0,0 +1,207 @@
+rancher:
```

Comment:
it would be nice to override or add to this from a jenkins argument (in the jenkins-job-builder side of things) but given the nature of this jenkinsfile, I understand that may not be possible as there are already other args that must be hard coded like this.